test/CheckoutSdk.Tests/ValidationTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Checkout.Sdk.Payments;
+using Shouldly;
+using Xunit;
+
+namespace Checkout.Sdk.Tests
+{
+    public class ValidationTests
```

Comment:
Should be `CustomerSourceTests`

Reply:
CustomerSourceTests is another class that contains integration tests. Should these two be mixed?